### PR TITLE
Feature/misc bugs4

### PR DESCRIPTION
**Pull request by Lyncee59**
## Description
- New component calendar
- Warning messages for SendBTC fee
- Exchange history modal (BTC and BCH) related tickets 
- Upgrade redux-form 6.8.0 -> 7.3.0

## Change Type
Please enter one or more of the following: 
- Feature
- Bug Fix
- Upgrades

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.

## PR Creator Checklist
- [x] Code compiles correctly
- [x] No lint issues exist (verified via `yarn lint`)
- [ ] All unit tests pass (verified via `yarn test`)
- [x] Updated `README.md` and other documentation (if necessary)

## PR Reviewer Checklist
- [ ] Change validated and application spot checked
- [ ] Code styles and best practices met
- [ ] Code is readable and commented when necessary

### CODE REVIEW

File: packages/blockchain-info-components/src/Form/DateInput/index.js [hunk 1]
```diff
@@ -168,7 +169,7 @@ const BaseDateInput = styled(ReactDatetime)`
   .rdtPicker th.rdtDisabled,
   .rdtPicker th.rdtDisabled:hover {
     background: none;
-    color: ${props => props.theme['gray-2']};
+    color: ${props => props.theme['gray-1']};
     cursor: not-allowed;
```
Comment (plondon) [inline packages/blockchain-info-components/src/Form/DateInput/index.js]: Do we mind all the other hex codes in here?

**Comment by plondon** [comment]
Date component doesnt work for me:

https://gyazo.com/e9bcf82b3a473a10862321b00bd47d41

Error:

```Uncaught TypeError: Cannot read property 'type' of null
    at getValue (getValue.js:29)
    at onChangeValue (onChangeValue.js:14)
    at Object.ConnectedField._this.handleChange [as onChange] (ConnectedField.js:112)
    at eval (index.js:82)```

**Comment by plondon** [comment]
Autologout button does not click to allow me to change the amount.

**Comment by schnogz** [comment]
> - Upgrade redux-form 6.8.0 -> 7.3.0
👏 🙌

File: packages/blockchain-info-components/src/Form/DateInput/index.js (hunk 1 above)
Comment (Lyncee59) [reply packages/blockchain-info-components/src/Form/DateInput/index.js]: there is a few left yes, but are not very relevant I think (border-shadows etc...)

Comment (Lyncee59) [reply packages/blockchain-info-components/src/Form/DateInput/index.js]: I mean I tried switching the theme and I did not see anything shocking :)

**Pull request merged**